doc/doxygen/generate-changelog.py

```diff
@@ -16,7 +16,7 @@ def generate_changelog(template_filename, changelog_filename, output_filename):
          open(changelog_filename) as changelog, \
          open(output_filename, "w") as output:
         changelog_lines = []
-        release_title = re.compile(r"((RIOT-\d{4}\.\d{2} - Release Notes)|(Release 2013\.08))")
+        release_title = re.compile(r"((RIOT-\d{4}\.\d{2}(.\d+)? - Release Notes)|(Release 2013\.08))")
```

Comment:
Should you also escape the `.` before `\d+` ?

```suggestion
        release_title = re.compile(r"((RIOT-\d{4}\.\d{2}(\.\d+)? - Release Notes)|(Release 2013\.08))")
```

(same comment for the other regexp change).

Reply:
True, will fix!

Reply:
Done